coderd/workspaceapps/apptest/setup.go

```diff
@@ -309,15 +309,31 @@ func createWorkspaceWithApps(t *testing.T, client *codersdk.Client, orgID uuid.U
 		},
 	}, workspaceMutators...)
 
+	// Intentionally going to choose a port that will never be chosen.
+	// Ports <1k will never be selected. 396 is for some old OS over IP.
+	// It will never likely be provisioned. Using quick timeout since
+	// it's all localhost
+	fakeAppURL := "http://127.1.0.1:396"
```

Comment:
AFAIK ports <1024 are privileged.
Does the test suite run as root?

Reply:
They do not. We do not want a listener on this port. So the fact they are privileged is what I am hoping for :+1:

Reply:
Essentially we want a port that will never be listened on in our tests. If we listen on port 0, then read the port and close the listener, there's a potential that another test will take that port that we hope nothing will listen on.

Using a port < 1024 and on an uncommon app as the URL for a "not running" app should serve our use case here. We're just dialing it to make sure it's not in use and then failing fast if it is.

Reply:
Nice, thanks for the explanation 👍